doc/editor/vim.md

```diff
@@ -37,10 +37,11 @@
         --preset mini \
         -G Ninja \
         -DCMAKE_TOOLCHAIN_FILE=../cmake/GccArmNoneEabi.cmake \
+	-DCMAKE_EXPORT_COMPILE_COMMANDS=TRUE
```

Comment:
Missing trailing `\ `

Also, you don't need to mention the CMAKE_EXPORT_COMPILE_COMMANDS in the text below anymore.